pkg/upgrade/upgrade.go

```diff
@@ -33,5 +36,10 @@ func RunUpgrade(tasks ...Task) error {
 			return fmt.Errorf("upgrade failed. Error : %v", err)
 		}
 	}
+	alertlog.Logger.Infow("",
+		"eventcode", "ndm.upgrade.success",
+		"msg", "Successfully upgraded node disk manager",
```

Comment:
We are actually upgrading only the custom resources as part of this. If needed the message can be changed.

Reply:
I guess, we need to change the function name itself to state what it is doing.

Reply:
Also, the log message.

Reply:
Will take up in the next PR.